README.md

```diff
@@ -1,34 +1,30 @@
-# EIPs [![Gitter](https://badges.gitter.im/Join%20Chat.svg)](https://gitter.im/ethereum/EIPs?utm_source=badge&utm_medium=badge&utm_campaign=pr-badge)
+# Ethereum Improvement Proposals (EIPs) [![Gitter](https://badges.gitter.im/Join%20Chat.svg)](https://gitter.im/ethereum/EIPs?utm_source=badge&utm_medium=badge&utm_campaign=pr-badge)
+
 Ethereum Improvement Proposals (EIPs) describe standards for the Ethereum platform, including core protocol specifications, client APIs, and contract standards.
 
-A browsable version of all current and draft EIPs can be found on [the official EIP site](http://eips.ethereum.org/).
+**Before you initiate a pull request**, please read the process document. Ideas should be thoroughly discussed on the [Ethereum Magicians forums](https://ethereum-magicians.org) first.
```

Comment:
There has been pushback in the past about "forcing" people to use ethereum-magicians forum.  Consider weakening the language here to something like:
> Before you initiate a pull request, please read the process document.  Ideas should be thoroughly discussed prior to opening a pull request, such as on the Ethereum Magicians forums or in a GitHub issue in this repository.

Reply:
Addressing the extreme quantity of low quality issues in this repo is still an issue. (Solution: close GitHub issues and refer to Ethereum Magicians.) But if that will slow this PR down, then I am happy to do it separately.

Updated in 6abedf8

Reply:
Does this update address the issue?